### PR TITLE
Use community display name in cross-post listings

### PR DESCRIPTION
**Pull request by rscmbbng**
Rather than the AP actor name.

### CODE REVIEW

File: src/shared/components/post/post-listing.tsx [hunk 1]
```diff
@@ -498,9 +498,11 @@ export class PostListing extends Component<PostListingProps, PostListingState> {
             {dupes.map(pv => (
               <li className="list-inline-item mr-2">
                 <Link to={`/post/${pv.post.id}`}>
-                  {pv.community.local
-                    ? pv.community.name
-                    : `${pv.community.name}@${hostname(pv.community.actor_id)}`}
+                  {pv.community.local ? (
+                    <CommunityLink community={pv.community} />
```
Comment (dessalines) [inline src/shared/components/post/post-listing.tsx:502]: Look above. CommunityLink is already a link, so don't wrap it in another <Link..

**Review by dessalines — CHANGES_REQUESTED** [review]

File: src/assets/css/themes/_variables.news.scss [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+$info: $blue;
+$body-bg: #fff;
+$success: $indigo;
+$danger: darken($primary, 25%);
+$navbar-light-hover-color: $gray-900;
+$card-bg: $gray-100;
+$border-color: $gray-700;
+$mark-bg: rgb(255, 252, 239);
+$font-weight-bold: 600;
+$rounded-pill: 0.25rem;
```
Comment (dessalines) [inline src/assets/css/themes/_variables.news.scss:32]: Why is this checked in?

File: src/shared/components/home/home.tsx [hunk 3]
```diff
@@ -280,87 +280,15 @@ export class Home extends Component<any, HomeState> {
         />
         {this.state.siteRes.site_view?.site && (
           <div class="row">
-            <main role="main" class="col-12 col-md-8">
-              <div class="d-block d-md-none">{this.mobileView()}</div>
+            <main role="main" class="col-12">
```
Comment (dessalines) [inline src/shared/components/home/home.tsx:283]: Why are you removing the mobile view?

File: src/shared/components/post/post-listing.tsx [hunk 4]
```diff
@@ -327,30 +351,16 @@ export class PostListing extends Component<PostListingProps, PostListingState> {
...
-                {hostname(post_view.post.url)}
-              </a>
-            </li>
-            <li className="list-inline-item">•</li>
-          </>
-        )}
         <li className="list-inline-item">
           <span>
-            <MomentTime data={post_view.post} />
+            <MomentTime data={post_view.post} /> ago
```
Comment (dessalines) [inline src/shared/components/post/post-listing.tsx:356]: i18n strings, also did you look at the MomentTime component? Tell me the props on it.

File: src/shared/components/post/post-listing.tsx [hunk 5]
```diff
@@ -1121,16 +1130,15 @@ export class PostListing extends Component<PostListingProps, PostListingState> {
...
         <div class="d-none d-sm-block">
           <div class="row">
-            {!this.props.viewOnly && this.voteBar()}
-            <div class="col-sm-2 pr-0">
+            {!this.props.viewOnly}
+            {/*            <div class="col-sm-2 pr-0">
               <div class="">{this.thumbnail()}</div>
-            </div>
-            <div class="col-12 col-sm-9">
+            </div>*/}
```
Comment (dessalines) [inline src/shared/components/post/post-listing.tsx:1136]: .

File: src/shared/components/post/post-listing.tsx [hunk 6]
```diff
@@ -295,13 +295,37 @@ export class PostListing extends Component<PostListingProps, PostListingState> {
...
+    );
+  }
+
   createdLine() {
     let post_view = this.props.post_view;
     return (
       <ul class="list-inline mb-1 text-muted small">
         <li className="list-inline-item">
+          <span>
+            <span class="mr-1">Posted by</span>
```
Comment (dessalines) [inline src/shared/components/post/post-listing.tsx:326]: i18n strings?

File: src/shared/components/post/post-listing.tsx [hunk 7]
```diff
@@ -535,27 +551,27 @@ export class PostListing extends Component<PostListingProps, PostListingState> {
...
+                    {this.reportButton}
+                    {this.blockButton}
+                  </>
+                )}
+                {this.myPost && (this.showBody || this.state.showAdvanced) && (
+                  <>
+                    {this.editButton}
+                    {this.deleteButton}
+                  </>
+                )}
```
Comment (dessalines) [inline src/shared/components/post/post-listing.tsx:572]: So much reorg with no explanation.

**Review by dessalines — CHANGES_REQUESTED** [review]
Mark as draft if you're not ready for me to review it, but also I don't know why there's so much else checked in here that have nothing to do with the change.

**Comment by rscmbbng** [comment]
> Mark as draft if you're not ready for me to review it, but also I don't know why there's so much else checked in here that have nothing to do with the change.

Apologies, I was not aware it would add any subsequent things I'd push to main to the PR! I'm making heavily customized client so I think the majority of the changes are not relevant to you. I'll address the changes to community display names and make a separate clean PR.

**Comment by dessalines** [comment]
No probs, thx!

**Pull request closed without merging**